packages/react-components/src/hooks/useAlarms/hookHelpers/useLatestAlarmPropertyValues.ts

```diff
@@ -47,6 +47,7 @@ export function useLatestAlarmPropertyValues({
     iotSiteWiseClient,
     requests: alarmPropertyRequests,
     enabled,
+    refreshRate: Infinity,
```

Comment:
This is fine for now, but I would have kept the setting for refreshRate passed through, and then in useAlarms set the refreshRate to Infinity. That way this hook doesn't hardcode fetching the latest for some property just once.